examples/notebooks/CfRadial1_Model_Transformation.ipynb

```diff
@@ -57,7 +57,7 @@
    "source": [
     "## Open CfRadial1 file using xr.open_dataset\n",
     "\n",
-    "Making use of the xarray `netcdf4` backend. We get back all data and metadata in one single CfRadial1 Dataset."
+    "Making use of the xarray `netcdf4` backend. We get back all data and metadata in one single CfRadial1 Dataset. Since xarray 2025.04.0 we have to use `decode_timedalte=False` to prevent erroneous decoding of timdelta values for eg. pulse widths."
```

Comment:
```suggestion
    "Making use of the xarray `netcdf4` backend. We get back all data and metadata in one single CfRadial1 Dataset. Since xarray 2025.04.0 we have to use `decode_timedalte=False` to prevent erroneous decoding of timedelta values for eg. pulse widths."
```